tla/consensus/ccfraft.tla

```diff
@@ -716,7 +712,7 @@ AppendEntries(i, j) ==
                \/ m.entries # <<>>
             /\ Send(m)
             \* Record the most recent index we have sent to this node.
-            \* (see https://github.com/microsoft/CCF/blob/9fbde45bf5ab856ca7bcf655e8811dc7baf1e8a3/src/consensus/aft/raft.h#L935-L936)
+            \* raft.h::send_append_entries
```

Comment:
Why not keep both?

Reply:
The line numbers go out of date, this one was too. Function/type/variable names are more stable. On the TLA side, I don't think we have an easy way to do this, but for the C++, we do produce XML via doxygen, used by Sphinx ([example](https://microsoft.github.io/CCF/main/build_apps/kv/api.html#map)). We could build a tool that parses some of the comments in the TLA file, and checks references (we could make up something like `\* @xref:raft.h:aft::Aft::send_append_entries()` or whatever other prefix that's reasonably unique), to look them up in the doxygen XML, and validate they exist to avoid breaking changes.